src/aws-ssm-session.ts

```diff
@@ -52,10 +54,35 @@ export default class AWSSSMSession {
 					this.outputMap.set(message.MessageId, text);
 					break
 				}
-				case 'input_stream_data':
-				case 'start_publication':
-				case 'pause_publication':
+				case 'start_publication': {
+					if ( this.paused === false ) {
+						console.log( 'Session recieved start_publication message when the connection was not paused.' );
```

Comment:
These console warnings may be problematic for output in Node contexts; do they occur often?

Reply:
No, these should be unusual. Now we also `emit` we could remove them though.